### DISCUSSION TITLE
Tic80 built-in Editor

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2196] [Ideas]
No, it's not what you "probably" thought.

It's more like a tip, variables like "true", "false", "null", "nil", "undefined", "NaN" and "infinity" should have the colors of the "NUMBER" value declared in config.tic

Because they really are numbers... I don't think it's right for "true" and "false" to be the same color as "function" and "new" (in red)

[Is my idea valid?]

**Comment by nesbox** [comment]
I think they are reserved words for a language, not numbers
